### ISSUE TITLE
Promotion in GUI causes deep_copy exception

### ISSUE THREAD

**Issue by cclauss** [671620616/PhantomChess#55]
Does promotion work when not in GUI mode?
In GUI mode it throws an exception which is usually tied to deep_copy()
http://stackoverflow.com/questions/12658824/python-objects-from-existing-objects-using-new

```
      New Game     
-------------------
  a b c d e f g h  
8 r n b q k b n r 8
7 p p p p p p p p 7
6   .   .   .   . 6
5 .   .   .   .   5
4   .   .   .   . 4
3 .   .   .   .   3
2 P P P P P P P P 2
1 R N B Q K B N R 1 <
  a b c d e f g h  
:> a2a4
:> b7b5
:> a4b5
:> b8a6
:> b5b6
:> h7h6
:> b6b7
:> g7g6
      New Game     
-------------------
  a b c d e f g h  
8 r . b q k b n r 8
7 p P p p p p .   7
6 n .   .   . p p 6
5 .   .   .   .   5
4   .   .   .   . 4
3 .   .   .   .   3
2   P P P P P P P 2
1 R N B Q K B N R 1 <
  a b c d e f g h  
:> gui
# in the GUI, move the pawn forward (b7b8) to kick off promotion process.
:> promoting Q
### !exc_catch: caught an unpassed exception - TypeError:
    object.__new__(ChessError) is 
 -not safe, use exceptions.Exception.__new__()
```

**Comment by 671620616** [comment]
Instead of `promoting Q`, try doing `promote b7 Q`.  I *think* thats how the Run_this.py file accepts promoting commands.

**Comment by 671620616** [comment]
That wasn't the issue - I messed up the MRO in `exceptions.py`.  Fixed in 30b61d3b08e06b94707946d96502d17c3e7b3b96.
